apps/scriptkit/src/components/navigation.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react'
 import Link from 'components/link'
 // import Logo from 'images/logo.svg'
 import {Disclosure} from '@headlessui/react'
-import {Bars3Icon, XMarkIcon} from '@heroicons/react/20/solid'
+import {ViewBoardsIcon, XIcon} from '@heroicons/react/solid'
```

Comment:
@vojtaholik all the other projects are a version behind with the icons so I did a best guess here.